### PR TITLE
Laravel 8.x Documentation with Ably Broadcaster

### PR DESCRIPTION
**Pull request by qsdstefan**
Add information regarding use of Ably Laravel Broadcaster and Ably Laravel Echo, add deprecated section using ably-pusher adapter

Fixes https://github.com/ably/ably-php-laravel/issues/32

### CODE REVIEW

File: broadcasting.md [hunk 1]
```diff
@@ -108,19 +109,31 @@ The [laravel-websockets](https://github.com/beyondcode/laravel-websockets) and [
...
 
-If you plan to broadcast your events using [Ably](https://ably.io), you should install the Ably PHP SDK using the Composer package manager:
+If you plan to broadcast your events using [Ably](https://ably.com), you should install the Ably Laravel Broadcaster using the Composer package manager:
 
-    composer require ably/ably-php
+```shell
+composer require ably/laravel-broadcaster
+```
+
+> Deprecated - Use this instead, if you are using pusher-js client to connect with ably.
```
Comment (owenpearson) [inline broadcasting.md:118]: I don't know if it's a great idea to mark the existing setup as deprecated. Maybe it's fine but I think just a paragraph or two at the top saying that the recommended way to use ably with laravel is to use ably/laravel-broadcaster and linking out to our own documentation.

File: broadcasting.md [hunk 2]
```diff
@@ -188,45 +201,122 @@ window.Echo = new Echo({
 <a name="client-ably"></a>
 ### Ably
 
-[Laravel Echo](https://github.com/laravel/echo) is a JavaScript library that makes it painless to subscribe to channels and listen for events broadcast by your server-side broadcasting driver. You may install Echo via the NPM package manager. In this example, we will also install the `pusher-js` package.
-
-You may wonder why we would install the `pusher-js` JavaScript library even though we are using Ably to broadcast our events. Thankfully, Ably includes a Pusher compatibility mode which lets us use the Pusher protocol when listening for events in our client-side application:
+[Ably Laravel Echo](https://github.com/ably-forks/echo/) is a JavaScript library that makes it painless to subscribe to channels and listen for events broadcast by your server-side broadcasting driver. Ably is maintaining a fork of the official laravel-echo module which allows you to use the official [ably-js](https://github.com/ably/ably-js) SDK. In this example, we will also install the official `ably` package:
```
Comment (owenpearson) [inline broadcasting.md:204]: I think calling it "Ably laravel echo" is a bit audacious, I think this paragraph should be structured more like:
> [Ably] maintain a fork of laravel echo which allows you to use the official [Ably JavaScript SDK]. In this example we will also install the official `ably` package:

**Review by owenpearson — CHANGES_REQUESTED** [review]

File: broadcasting.md (hunk 1 above)
Comment (sacOO7) [reply broadcasting.md:118]: Okay, this will make sense 👍

File: broadcasting.md (hunk 2 above)
Comment (sacOO7) [reply broadcasting.md:204]: Okay, I will update this accordingly.

Comment (sacOO7) [reply broadcasting.md:204]: I have updated code accordingly across different PRs.

**Comment by sacOO7** [comment]
Closing this PR with the introduction of #8 addressing above comments

**Pull request closed without merging**